modules/sdk/gradle-plugins-workspace/src/gradleTest/angularTest/build.gradle

```diff
@@ -0,0 +1,5 @@
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":clean"
```

Comment:
Can you update this test to have 3 types of modules
1) osgi portlet
2) hybrid java/js portlet (like this one has)
3) pure js-portlet

Then assert that 1 has LiferayOSGiplugin applied, 2 has LiferayOSGi applied, and 3) has FrontendPlugin applied

Reply:
Yes, I will work on that now.